channeldb/db.go

```diff
@@ -39,6 +39,9 @@ var (
 			// The base DB version requires no migration.
 			number:    0,
 			migration: nil,
+		}, {
+			number:    1,
+			migration: paymentStatusesMigration,
```

Comment:
This needs to be rebased and utilize the second migration slot as a PR recently landed that occupies this migration slot.